src/components/WelcomeScreen/Welcome.jsx

```diff
@@ -0,0 +1,78 @@
+import {Grid, CardMedia, Container, makeStyles,Button, Box } from '@material-ui/core';
+import React from 'react';
+import { useHistory } from 'react-router';
+
+const useStyles=makeStyles((theme)=>({
+    containerRoot: {
+        backgroundColor:'white',
+        padding: '32px 24px 100px 24px',
+      },
+    buttonRoot:{
+        width: '367px',
+        height: '50px',
+        fontSize:'12px',
+        marginTop:'20px', 
+        background: '#20D3C2'
+    }
+}))
+
+const WelcomePage = () => {
+    const classes = useStyles();
+    const history=useHistory()
+  return (
+    <>
+      <Container
+        className={classes.containerRoot}
+        maxWidth="sm"
+        disableGutters
+      >
```

Comment:
kalo bisa height nya dibikin full sesuai screen size, jadi gak ada space kosong di bawah
thank you

![image](https://user-images.githubusercontent.com/24664561/111820168-c5274080-8913-11eb-8993-1557f503a9c9.png)

Reply:
on progress resize height 320x726

Reply:
sudah oke mas ui nya, tolong diriview

Reply:
mas @bagasprakarso  pull request nya saya closed ya, karena belum termasuk ui yang diprioritaskan